### ISSUE TITLE
Add import/export function

### ISSUE THREAD

**Issue by TheBiostarfly** [apozas/contactdiary#8]
Hi thanks for this helpfull app. One feature that I'm missing is the possibility to export the contacts e.g. in case the health department request my contacts it would be handy if I could just export an CSV or something else. 
Thanks and keep up the great work!

**Comment by apozas** [comment]
Hello! Thanks for the suggestion. It is an interesting feature that can be useful for some, but I'm not very sure it will be actually useful for many. I am not aware of Health Departments being able to handle this data in a way other than manually. From what I have understood and how it works in my country, a contact tracer will call you and ask you by phone about your contacts. However, if you know about places that accept this form of notifying, it would be very helpful to me which do so.

In summary, I think it can be a useful addition in some cases, but currently it is low in the priority list.

**Comment by Torenga** [comment]
I would like this in combination with an import function, basically allowing backups and device migration.

**Comment by eutektoid** [comment]
In Germany it would be useful to give the authorities (Gesundheitsamt) a CSV or PDF export of the contact diary in case of positive testing for SARS-CoV-2. They can then integrate the data in their workflow without wasting precious time, talking to you on the phone, while you read your list to them. So instead they would directly contact your contacts.

I also like the import/export feature idea for a possibile device-migration scenario from @Torenga. So 👍 !
And thanks to @apozas  for your work! Can I in any way help with the development of the export functionality?

**Comment by apozas** [comment]
So, I understand that the Gesundheitsamt accepts CSV files for informing about your contacts? Any information/official announcement/documentation would be very much appreciated.

I've been unexpectedly busy these last weeks, but this is hight in the priority list. This weekend I want (hopefully) to make a release with a small change in the forms (see #4), and then go straight away to this. This being said, this is an open source project, and PRs are always appreciated :)

**Comment by eutektoid** [comment]
So as far as I know, there is a software called SORMAS® (Surveillance, Outbreak Response Management and Analysis System), which is developed here: https://github.com/hzi-braunschweig/SORMAS-Project, uses CSV files. According to coronika contact-diary app, it is used by Gesundheitsamt (https://www.coronika.app/ - https://github.com/kreativzirkel/coronika). Also in the financing report they write about it (https://www.bundesgesundheitsministerium.de/fileadmin/Dateien/3_Downloads/O/OEGD/SormasAtDEMIS_Projektsteckbrief.pdf) [in German].    
If I come across a template or formating guidelines for a CSV, I'll let you know!

**citation from coronika:**
> *  Export optimiert für Gesundheitsämter
> *  Übersicht kann als PDF oder CSV (SORMAS-kompatibel) einfach bearbeitet werden

**translation:**
 *  export opimized for Gesundheitsämter (plural of Gesundheitsamt)
 *  overview may be changed easily as PDF or CSV (compatible to SORMAS) 

Thank you for your work @apozas and thanks to everyone contributing to this project!

I am new to Android development and open source development in general, but I will try to do a PR in the near forseeable future. ;-)

**Comment by apozas** [comment]
Wow, this is useful information, indeed. I'll look it up to see how the export can be done best. Thank you very much!

**Comment by eutektoid** [comment]
You're welcome! By the way, I found a ["Guide to Global Digital Tools for COVID-19 Response"](https://www.cdc.gov/coronavirus/2019-ncov/global-covid-19/compare-digital-tools.html) by the CDC. This might be helpful to eventually offer different export options, depending on the country you're living in or software used by the contact tracers, respectively.

**Comment by apozas** [comment]
Well, seeing that there are not many problems after 1 week of the appearance on F-Droid, I'm closing this issue. There is the problem mentioned in #23, but that will be addressed there (and I think I know how to solve it already). Feel free to reopen if anything else, not covered in #23, comes up.
